src/features/campaigns/components/ActivityList/items/EmailListItem.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC } from 'react';
+import { EmailOutlined, Person } from '@mui/icons-material';
+
+import ActivityListItemWithStats from './ActivityListItemWithStats';
+import useEmail from 'features/emails/hooks/useEmail';
+import useEmailTargets from 'features/emails/hooks/useEmailTargets';
+import ActivityListItem, { STATUS_COLORS } from './ActivityListItem';
+import useEmailState, { EmailState } from 'features/emails/hooks/useEmailState';
+
+interface EmailListItemProps {
+  emailId: number;
+  orgId: number;
+}
+
+const EmailListItem: FC<EmailListItemProps> = ({ orgId, emailId }) => {
+  const { data: email } = useEmail(orgId, emailId);
+  const state = useEmailState(orgId, emailId);
+  const { data: targets } = useEmailTargets(orgId, emailId);
+
+  if (!email) {
+    return null;
+  }
+
+  let color = STATUS_COLORS.GRAY;
+  if (state === EmailState.SENT) {
+    color = STATUS_COLORS.GREEN;
+  } else if (state === EmailState.SCHEDULED) {
+    color = STATUS_COLORS.BLUE;
+  }
+
+  //TODO: use actual values for these:
+  const blueChipValue = 0;
+  const greenChipValue = 0;
+  const orangeChipValue = 0;
+
+  const hasStats = !!blueChipValue || !!greenChipValue || !!orangeChipValue;
```

Comment:
I'm not sure I agree about this check. It's fine for now, but I think it's better to check whether the email has been sent, i.e. whether the published date has passed. That way, we can still show 0/0/0, but only when those numbers are 0 because the email has been published but not delivered to anyone.

In practice, that should very rarely be the case.

Reply:
will change to look at publish date instead!